### PR TITLE
Fix some doc comments

### PR DESCRIPTION
**Pull request by srawlins**
Should ultimately lead to the fix of https://github.com/google/quiver-dart/issues/749, when quiver 3.2.2 is released.

### CODE REVIEW

File: lib/src/collection/multimap.dart [hunk 1]
```diff
@@ -23,9 +23,10 @@ abstract class Multimap<K, V> {
   /// Constructs a new list-backed multimap.
   factory Multimap() => ListMultimap<K, V>();
 
-  /// Constructs a new list-backed multimap. For each element e of [iterable],
-  /// adds an association from [key](e) to [value](e). [key] and [value] each
-  /// default to the identity function.
+  /// Constructs a new list-backed multimap.
+  ///
+  /// For each element `e` of [iterable], adds an association from [key] to
+  /// [value]. [key] and [value] each default to the identity function.
```
Comment (cbracken) [inline lib/src/collection/multimap.dart:29]: This is not what the constructor does. We're mapping from `key(e)` to `value(e)`.

**Review by cbracken — APPROVED** [review]
Added a note.

**Review by cbracken — CHANGES_REQUESTED** [review]
Sorry that should have been request changes.

File: lib/src/collection/multimap.dart (hunk 1 above)
Comment (srawlins) [reply lib/src/collection/multimap.dart]: Updated, thanks!

**Review by cbracken — APPROVED** [review]
LGTM stamp from a Japanese personal seal

Thanks!

**Pull request merged**